### ISSUE TITLE
Reviewers in Teams does not match Reviewers in GitHub

### ISSUE THREAD

**Issue by bradcutshall** [integrations/microsoft-teams#251]
**Describe the bug**
All "Reviewers" are removed in teams when a PR is completed. However, in GitHub, the reviewer is still present on the PR. It seems expected that the reviewer should still be listed in the card on teams even after the PR is completed, since they are still assigned as a reviewer on the now merged PR. It's preferred that the teams card match the data in GitHub.

**To Reproduce**
- Create a new PR, notice the card is created as a new thread in teams.
- Assign a user as a "Reviewer", notice the "Reviewers:" data on the card in teams is updated with the user added on GitHub.
- Complete a review and merge the PR, the card's status updates to merged, but the "Reviewers" is now showing "No one"

**Expected behavior**
"Reviewers:" should match users assigned as reviewers in GitHub.

**Screenshots**
![image](https://user-images.githubusercontent.com/110041485/197058564-0bbee970-718c-454c-8dce-b21ddad32476.png)

**Desktop (please complete the following information):**
 - OS: macOS Monterey version 12.6
 - Version: 1.5.00.26358

**Comment by aasiddiq** [comment]
@bradcutshall thank you for raising this issue. We have identified the issue and will push a fix soon for this.

**Comment by aasiddiq** [comment]
@bradcutshall Fix is now live. If you see any further issue feel free to reopen.

**Comment by bradcutshall** [comment]
We ran into an issue with the reviewers. Two reviewers were added to the PR, and in teams it shows the two reviewers, plus the person who created the PR. However, where it gets more weird, the two reviewers and the person who created the PR were all added four more times. 

For example in GitHub three users were involved, the user who created the PR and users A and B who were assigned as reviewers. On Teams, in the reviewers section, it showed: "Reviewers A, PR, A, A, A, A, A, B, PR, PR, PR, A".
